python/lsst/sims/maf/bundles/glanceBundle.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import print_function
+from builtins import zip
+import lsst.sims.maf.metrics as metrics
+import lsst.sims.maf.slicers as slicers
+import lsst.sims.maf.stackers as stackers
+import lsst.sims.maf.plots as plots
+import lsst.sims.maf.metricBundles as metricBundles
+import lsst.sims.maf.utils as utils
+
+__all__ = ['glanceBundle']
+
+
+def glanceBundle(colmap_dict=None, nside=64):
+    """Generate a handy set of metrics that give a quick overview of how well a survey performed
```

Comment:
Maybe add a 'validate_colmap_dict' method (per bundle, like the glanceBundle) that checks all columns used in the glanceBundle are in the dictionary?